package.json

```diff
@@ -20,9 +19,11 @@
     "url": "https://github.com/treasure-data/td-client-node.git"
   },
   "license": "Apache-2.0",
-  "main": "./lib",
+  "main": "./dist/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "npm run build && ./node_modules/.bin/mocha",
+    "build": "babel lib -d dist",
+    "prepublish": "npm run build"
   },
   "config": {
     "blanket": {
```

Comment:
nit: this line will not be used anymore.

Reply:
👍